source/api/presentation/3.0/index.md

```diff
@@ -82,23 +82,30 @@ Collectively, the use cases require a model in which one can characterize the ob
 
 ### 1.3. Terminology
 
-The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _SHOULD NOT_, _RECOMMENDED_, _MAY_, and _OPTIONAL_ in this document are to be interpreted as described in [RFC 2119][rfc-2119].
-
-This specification also uses the following terms:
+This specification uses the following terms:
```

Comment:
Good point, I think it might be helpful to add another introductory statement saying that we use the JSON-LD terminology which would give that context. Could do something like:

> This specification uses the terminology of the [JSON-LD specification](https://www.w3.org/TR/json-ld/) to describe the serialization of resources. This specification also uses the following terms:

Reply:
We don't really use the terminology of JSON-LD, just of JSON (RFC7159).  E.g. in https://json-ld.org/spec/latest/json-ld/#terminology we use the first section, but not really the second LD specific ones other than "context" Happy to add in the 7159 link though.

Reply:
Doesn't need to be JSON-LD specific, just that "embedded" means embedded `{...}` rather than some other notion of embedded.

Reply:
Yes, I guess we do mostly use JSON and not JSON-LD so that would be the better link

Reply:
Fixed.